#!/bin/bash
#SBATCH --job-name=data-proc
#SBATCH --partition=gpu-2080ti
#SBATCH --account=cse
#SBATCH --nodes=1
#SBATCH --cpus-per-task=8
#SBATCH --mem=32G
#SBATCH --gres=gpu:1
#SBATCH --time=12:00:00 
#SBATCH --mail-type=ALL
#SBATCH --mail-user=tjung2@uw.edu

# I use source to initialize conda into the right environment.
cat $0
echo "--------------------"

source ~/.bashrc
conda activate ckl

# python run.py --config configs/templama/training/t5_kadapters_yearly_2freeze.json -checkpoint_path outputs/wmtbaseline_full/epoch=0-f1_score=0.151-em_score=0.000.ckpt

# python run.py --config configs/wmt/training/t5_kadapters_yearly_2freeze.json -checkpoint_path outputs/wmtkadapter_2010_2freeze_158_128/epoch=8-f1_score=0.084-em_score=0.000.ckpt

python preprocess_wmt_train_data.py 2010
python preprocess_wmt_train_data.py 2011
python preprocess_wmt_train_data.py 2012--------------------
0
/mmfs1/gscratch/ark/tjung2/miniconda3/envs/ckl/lib/python3.8/site-packages/transformers/pipelines/base.py:908: UserWarning: You seem to be using the pipelines sequentially on GPU. In order to maximize efficiency please use a dataset
  warnings.warn(
Traceback (most recent call last):
  File "preprocess_wmt_train_data.py", line 162, in <module>
    main()
  File "preprocess_wmt_train_data.py", line 73, in main
    sentence, answers = mask_sentence(sentences[i], mask_chance, bert_res[i], spacy_res[i])
IndexError: list index out of range
0
/mmfs1/gscratch/ark/tjung2/miniconda3/envs/ckl/lib/python3.8/site-packages/transformers/pipelines/base.py:908: UserWarning: You seem to be using the pipelines sequentially on GPU. In order to maximize efficiency please use a dataset
  warnings.warn(
Traceback (most recent call last):
  File "preprocess_wmt_train_data.py", line 162, in <module>
    main()
  File "preprocess_wmt_train_data.py", line 73, in main
    sentence, answers = mask_sentence(sentences[i], mask_chance, bert_res[i], spacy_res[i])
IndexError: list index out of range
0
/mmfs1/gscratch/ark/tjung2/miniconda3/envs/ckl/lib/python3.8/site-packages/transformers/pipelines/base.py:908: UserWarning: You seem to be using the pipelines sequentially on GPU. In order to maximize efficiency please use a dataset
  warnings.warn(
slurmstepd: error: *** JOB 6242334 ON z3001 CANCELLED AT 2022-09-23T03:26:53 ***
